src/CoreBundle/private/library/classes/dbobjects/TCMSRecordWritable.class.php

```diff
@@ -410,11 +410,11 @@ public function LoadFromRow($aRow, $bConvertFieldsToSQL = false)
     public function ConvertToSQLFormat($aData = null)
     {
         if (is_null($aData)) {
-            $aData = &$this->sqlData;
+            $aData = $this->sqlData;
```

Comment:
`$aData`wird weiter unten modifiziert. Ich bin mir nicht ganz sicher, wie sich das verhält, wenn man da ein Attribut ohne Referenz zuweist, aber ich fürchte, die Änderung kommt dann nicht in `$this->sqlData` an. Davor war das aber so.

Reply:
this should not be a problem - as we return the value and pass it to LoadFromRow which will update sqlData